playwright.config.ts

```diff
@@ -1,21 +1,22 @@
-/* eslint-disable import/no-extraneous-dependencies */
-/* eslint-disable import/no-import-module-exports */
+/* eslint-disable @typescript-eslint/no-empty-function */
+/* eslint-disable @typescript-eslint/no-var-requires */
+
 import { defineConfig } from '@playwright/test';
-import { isEmpty, toNumber } from 'lodash';
+import { toNumber } from 'lodash';
 
 const useSessionReporter = true;
+require('source-map-support').install = () => {};
```

Comment:
can you add a comment before that line saying
```suggestion
// NOTE: without this, the wrong source map is loaded and the stacktraces are all wrong 
// see https://github.com/microsoft/playwright/issues/29157#issuecomment-1915000626
require('source-map-support').install = () => {};
```